### PR TITLE
Improve done and fail messages and workaround for breaking progressbar

### PR DESCRIPTION
**Pull request by nelson6e65**
- [x] Add workaround for maximum steps counter progress :bug: which break the progress-bar.
- [x] Add :new: message for failing state.
- [x] Improve the "finished" message.
- [x] :art: Add colours for success and fail messages.

### CODE REVIEW

File: src/Console/Helper/ProgressBar.php [hunk 1]
```diff
@@ -41,8 +41,18 @@ public function increment(int $increment = 1): void
         }
 
         $this->bar->advance($increment);
-        if ($this->bar->getProgress() === $this->bar->getMaxSteps()) {
```
Comment (TomasVotruba) [inline src/Console/Helper/ProgressBar.php]: I think better solution would be in this condition to stop it of overload. I'd try something like:

```php
if ($this->bar->getProgress() >= $this->bar->getMaxSteps()) {
}
```

Comment (nelson6e65) [inline src/Console/Helper/ProgressBar.php]: The #902 had this 5 "extra steps". This is not a problem about the maximum is set in the [`init()`](https://github.com/ApiGen/ApiGen/blob/ad95bd33bb2869bed7f19061ab88d092389926a1/src/Console/Helper/ProgressBar.php#L30)?

#905

**Comment by vlastavesely** [comment]
After #927 gets merged, progressbar issue should be fixed for ever and ever. I hope :)

**Comment by TomasVotruba** [comment]
Replaced by #927 

Thank you for your work anyway!

**Pull request closed without merging**